File: src/SOP8/Egovframework/Base/base-account-ibll/src/main/java/egovframework/base/account/ibll/response/ResponseGradeList.java
package egovframework.base.account.ibll.response;

import java.util.ArrayList;
import java.util.List;
import egovframework.base.model.account.Grade;
import egovframework.base.response.MessageResult;

public class ResponseGradeList extends MessageResult {

    private List<Grade> grades = new ArrayList<>();

    public List<Grade> getGrades() {
        return grades;
    }

    public void setGrades(List<Grade> grades) {
        this.grades = grades;
    }

    public ResponseGradeList() {
        super();
    }

    public ResponseGradeList(boolean success, String message) {
        super(success, message);
    }
}

File: src/SOP8/Base/Account/Base.Account.IBLL/Response/ResponseGradeList.cs
using Response;
using System.Collections.Generic;
using Base.Model.Account;

namespace Base.Account.IBLL.Response
{
    public class ResponseGradeList : MessageResult
    {
        private List<Grade> m_grades = new List<Grade>();

        public List<Grade> Grades
        {
            get { return m_grades; }
            set { m_grades = value; }
        }

        public ResponseGradeList()
            : base()
        {
        }

        public ResponseGradeList(bool success, string message)
            : base(success, message)
        {
        }
    }
}
